manta-trusted-setup/src/bin/groth16_phase2_client.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019-2022 Manta Network.
+// This file is part of manta-rs.
+//
+// manta-rs is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// manta-rs is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with manta-rs.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Trusted Setup Ceremony Client
+
+extern crate alloc;
+
+use clap::{Parser, Subcommand};
+use dialoguer::{theme::ColorfulTheme, Input};
+use manta_trusted_setup::groth16::ceremony::{
+    config::ppot::{client_contribute, display_on_error, get_client_keys, register, Config},
+    CeremonyError,
+};
+use manta_util::Array;
+
+/// Welcome Message
+pub const TITLE: &str = r" __  __             _          _____               _           _
+|  \/  | __ _ _ __ | |_ __ _  |_   _| __ _   _ ___| |_ ___  __| |
+| |\/| |/ _` | '_ \| __/ _` |   | || '__| | | / __| __/ _ \/ _` |
+| |  | | (_| | | | | || (_| |   | || |  | |_| \__ \ ||  __/ (_| |
+|_|  |_|\__,_|_| |_|\__\__,_|   |_||_|   \__,_|___/\__\___|\__,_|
+
+ ____       _
+/ ___|  ___| |_ _   _ _ __
+\___ \ / _ \ __| | | | '_ \
+ ___) |  __/ |_| |_| | |_) |
+|____/ \___|\__|\__,_| .__/
+                     |_|
+";
+
+/// Command
+#[derive(Debug, Subcommand)]
+pub enum Command {
+    /// Register for the Trusted Setup Ceremony
+    Register,
+
+    /// Runs the Trusted Setup Ceremony as a Contributor
```

Comment:
```suggestion
    /// Contribute to the Trusted Setup Ceremony as a Contributor
```

Reply:
No, I think the original comment is better.